grails-app/conf/kuorum/CustomRegisterFilterFilters.groovy

```diff
@@ -32,27 +32,5 @@ class CustomRegisterFilterFilters {
 
             }
         }
-
-        configFomain(controller: 'admin|logout|error|register|layouts|file|login', invert:true){
-            before = {
-                if (springSecurityService.isLoggedIn() && SpringSecurityUtils.ifAnyGranted("ROLE_ADMIN")){
-                    DomainRSDTO domainRSDTO = CustomDomainResolver.domainRSDTO
-                    if (domainRSDTO.version == 0){
-//                      Domain is not configured
-                        log.info("Domain is not configured. Redirecting ...")
-                        redirect(mapping: 'adminDomainRegisterStep1')
-
-                        return false
-                    }
-                }
-
-            }
-            after = { Map model ->
-
-            }
-            afterView = { Exception e ->
-
-            }
-        }
```

Comment:
If you delete all this code, then the class has no sense. So please remove it, and check that all runs properly.